addons/mass_mailing/static/tests/tours/mailing_editor.js

```diff
@@ -39,7 +39,7 @@ registry.category("web_tour.tours").add('mailing_editor', {
     },
 }, {
     content: 'wait for the snippet menu to finish the drop process',
-    trigger: 'body:not(:has(.o_we_already_dragging))',
+    trigger: 'body:not(:has(.o_we_ongoing_insertion))',
```

Comment:
Commit message:
- Blank line after title... otherwise you break git tools
- Missing `*: ` before list of apps